ssl/t1_lib.c

```diff
@@ -343,6 +343,7 @@ static int add_provider_groups(const OSSL_PARAM params[], void *data)
      * it.
      */
     ret = 1;
+    (void)ERR_set_mark();
```

Comment:
We do not use the void cast elsewhere when calling ERR_set_mark() and other mark functions.

Reply:
We do it at least in several places, but it's not a problem to remove it

Reply:
Fixed, thanks